packages/components/src/components/hds/popover-primitive/index.hbs

```diff
@@ -0,0 +1,44 @@
+{{! @glint-nocheck: not typesafe yet }}
+{{!
+  Copyright (c) HashiCorp, Inc.
+  SPDX-License-Identifier: MPL-2.0
+}}
+<div class={{this.classNames}} ...attributes {{this.setupPrimitiveContainer}}>
+  {{#if this.toggleContainsInteractive}}
+    <div
+      class="hds-popover-primitive__toggle"
+      aria-controls={{this.popoverId}}
+      aria-describedby={{this.popoverId}}
+      aria-details={{this.popoverId}}
+      {{this.setupPrimitiveToggle}}
+    >
+      {{yield (hash popoverElement=this.popoverElement isOpen=this.isOpen) to="toggle"}}
+    </div>
+  {{else}}
+    <button
+      type="button"
+      class="hds-popover-primitive__toggle"
+      aria-controls={{this.popoverId}}
+      aria-describedby={{this.popoverId}}
+      aria-details={{this.popoverId}}
```

Comment:
Question: why do you have both `aria-describedby` and `aria-details` with the same values used here?

Reply:
I _think_ I have found a code example of a tooltip using it (but I can't find it anymore, apparently). Reading this https://www.scottohara.me/blog/2018/11/08/aria-details.html my understanding was that `aria-details` provides an extra layer of information. If that's not the case, I can remove one of the two (which one, in that case?).

Reply:
Ahh okay. We can safely remove `aria-details`, then.

Reply:
done dc10306062651c106f8c22d71e79280a2ae98081